cmd/object-handlers.go

```diff
@@ -88,6 +90,43 @@ func (f funcToWriter) Write(p []byte) (int, error) {
 	return f(p)
 }
 
+// HTTPWriteCloser writes to the underlying io.Writer
+// and also writes nil to the underlying io.Writer when
+// Close is called and no data was written.
+//
+// In case of a http.ResponseWriter the Close will set the
```

Comment:
a --> an

Reply:
Really? Yes it sounds better but I thought 'an' just before vocals?!

Reply:
No, it is a bit more complicated than that, so also for 'silent' consonants, see http://english.blogoverflow.com/2011/11/articles-a-vs-an/

Reply:
Oh, okay thanks